modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -425,8 +425,8 @@ const setup = (editor: Editor): RenderInfo => {
       OuterContainer.focusToolbar(outerContainer);
     });
 
-    editor.addCommand('ToggleToolbarDrawer', () => {
-      OuterContainer.toggleToolbarDrawer(outerContainer);
+    editor.addCommand('ToggleToolbarDrawer', (_ui, options?: { skipFocus: boolean }) => {
+      OuterContainer[options?.skipFocus ? 'toggleToolbarDrawerWithoutFocusing' : 'toggleToolbarDrawer'](outerContainer);
```

Comment:
This is a really confusing code style, please just reference the methods directly. In most cases (I realise not this one) this sort of thing actually breaks tree shaking, because we use `import * as` which doesn't actually create the object it just simulates it at compile time. Unless code like this is used; then the object is created.

- [ ] fix

Reply:
Updated [here](https://github.com/tinymce/tinymce/pull/8272/commits/d7dac2d8efe212d68eb2c0911f9b300a9dc37386).